scripts/fast-reboot

```diff
@@ -713,6 +713,11 @@ then
   systemctl stop "$service_name"
 fi
 
+if [[ "$sonic_asic_type" = 'clounix' ]];
```

Comment:
>clounix [](http://example.com/codeflow?start=28&length=7)

I could not find the asic type in other places. What is it?

Reply:
Hi, Qi, 
Clounix is a new ASIC platform. Thanks.